client.go

```diff
@@ -860,6 +904,14 @@ func NewClient[TTx any](driver riverdriver.Driver[TTx], config *Config) (*Client
 			client.services = append(client.services, pluginPilot.PluginServices()...)
 		}
 
+		// It's important for queuesIncluded to be `nil` in case it's not in use
+		// for the various driver queries to work correctly.
```

Comment:
maybe the driver layer should handle `nil` and `[]string{}` equivalently to avoid needing to deal with that concern at this level?